garminconnect/__init__.py

```
@@ -716,14 +716,56 @@ def get_personal_record(self) -> Dict[str, Any]:
 
         return self.connectapi(url)
 
-    def get_earned_badges(self) -> Dict[str, Any]:
+    def get_earned_badges(self) -> List[Dict[str, Any]]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)